extensions/qxsck/var-and-list.js

```diff
@@ -12,14 +12,22 @@
       clearList: '清空列表 [LIST]',
       deleteOfList: '删除列表 [LIST] 的第 [INDEX] 项',
       addValueInList: '在列表 [LIST] 末尾添加 [VALUE]',
+      addListToList: '在列表 [LIST] 末尾添加 [LIST2]',

```

Comment:
To reduce ambiguity I changed the English string to be `add json [LIST2] to [LIST]`; can you update the translation accordingly?

Reply:
> 为了减少歧义，我将英语字符串更改为 ;你能相应地更新翻译吗？`add json [LIST2] to [LIST]`

LIST2 is a list rather than JSON, JSON isn't suitable.

Reply:
having two separate blocks (addValueInList, addListToList) with the same text is untenable, so they will have to be distinguished somehow

Reply:
> 具有两个具有相同文本但不同行为的单独块（addValueInList，addListToList）是站不住脚的，因此必须以某种方式区分它们。你有更好的建议吗？

can replace ‘json’ to ‘list’?